app/src/main/java/org/simlar/https/HttpsPost.java

```diff
@@ -98,6 +99,8 @@ public static HttpsURLConnection createConnection(final String urlPath, final bo
 			return connection;
 		} catch (final MalformedURLException e) {
 			Lg.ex(e, "MalformedURLException");
+		} catch (final ProtocolException e) {
```

Comment:
Maybe
`} catch (final MalformedURLException | ProtocolException | IOException e) {`
`  			Lg.ex(e, "Failed to create connection for", urlPath);`